test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/npd-binding.yaml

```diff
@@ -0,0 +1,16 @@
+# This is the role binding for the node-problem-detector
+# which is used to get and patch the node health status.
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: node-problem-detector-binding
+  labels:
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Yes, it mostly just does gets and patches on node statuses. In fact there is an ongoing issue for npd RBAC -> https://github.com/kubernetes/node-problem-detector/issues/61
@Random-Liu can answer this question better.